houston/ardu/sandbox.py

```diff
@@ -281,6 +281,7 @@ def run_and_trace(self,
         timeout_command = 300 / speedup + 5
         timeout_arm = 10 / speedup + 5
         timeout_mission_upload = 20
+        do_delay = max(4, int(20 / speedup))  # seconds
```

Comment:
Do you mind adding a single-line comment above this line that describes how this delay is used?